Backend/controller/Auth.js

```diff
@@ -44,15 +44,22 @@ const auth = (req, res, next) => {
  * @param {*} userId The id of the user for which the token is created.
  * @returns A access token.
  */
-const create = (userId) => {
-  const token = jwt.sign({ id: userId }, process.env.JWT_LOGIN_TOKEN, {
-    expiresIn: "10m",
-  });
+const create = (userId, username) => {
+  const token = jwt.sign(
+    { id: userId, username: username },
```

Comment:
Die Erstellung des Objectes in eine eigene Methode auslagern. Dann kann man das in Zeile "createRefresh" wiederverwenden. Über den Methodennamen wäre dann auch die Zuordnung zum "User"-Object in jwt.verify evtl erleichtert.

Reply:
für eine Zeile eine eigene Methode?

Reply:
Wenn es die Verständlichkeit des Code erhöht ist das vollkommen legitim. 

Es wird ja an zwei Stellen benutzt. Über den Zugriff über eine Methode könnte man sicherstellen, dass stets ein gleichartiges Objekt erzeugt wird.